src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -42,7 +42,7 @@ class CallActor(key: CallKey, locallyQualifiedInputs: CallInputs, backend: Backe
 
   val call = key.scope
   val callReference = CallReference(workflowDescriptor.name, workflowDescriptor.id, call.fullyQualifiedName)
-  val tag = s"CallActor [$callReference]"
+  val tag = s"CallActor [$callReference${key.index.map(i => s" (shard $i)").getOrElse("")}]"
```

Comment:
per our standards there shouldn't be this much stuff happening in a string interpolation